crates/viewer/re_data_ui/src/entity_db.rs

```diff
@@ -101,6 +102,58 @@ impl crate::DataUi for EntityDb {
                 ui.end_row();
             }
 
+            {
+                let &ChunkStoreConfig {
+                    enable_changelog: _,
+                    chunk_max_bytes,
+                    chunk_max_rows,
+                    chunk_max_rows_if_unsorted,
+                } = self.store().config();
+
+                ui.grid_left_hand_label("Compaction");
```

Comment:
Maybe this should be called something like Compaction Policy?

Compaction initial reads as "How compacted is the store" and not "How does the store handle compaction"

Reply:
I agree. "Compaction threshold" also works

Reply:
> I agree. "Compaction threshold" also works

That's what I had in the beginning, but it made this column name the longest one, which made every the whole column larger, which introduced quite a lot of dead whitespace :sob: 

Similar problem I had when I added the `Source RRD version` thing...

I'll try `Compaction Policy`. If that doesn't work I'll make another the PR with a longer name and bring Katya into the discussion.